cns/api.go

```diff
@@ -335,3 +338,8 @@ type NmAgentSupportedApisResponse struct {
 	Response      Response
 	SupportedApis []string
 }
+
+type GetHomeAzResponse struct {
+	Response       Response           `json:"Response"`
+	HomeAzResponse nma.HomeAzResponse `json:"HomeAzResponse"`
```

Comment:
JSON keys should not be capitalized

Reply:
further, unless you don't want your JSON keys to match your exported var names for some reason, you don't need explicit `json` struct annotations at all.

Reply:
per the previous comment

> Please add a json struct tag here. Even if it's identical to the field name, it serves as documentation and it also breaks the binding between the field name and the JSON representation.

cc @timraymond

Reply:
okay, if we want to be explicit and always have the JSON annotations that's fine. They still should not start with a capital letter 🙂